sklearn/linear_model/_glm/glm.py

```diff
@@ -285,9 +285,10 @@ def fit(self, X, y, sample_weight=None):
                 jac=True,
                 options={
                     "maxiter": self.max_iter,
+                    "maxls": 40,  # default is 20
                     "iprint": (self.verbose > 0) - 1,
```

Comment:
Why not pass `self.verbose - 1` directly to allow printing more detailed information from the lfbgs solver?

Could be tackled in a separate PR if you prefer not to mix concerns.

Reply:
Taken with pleasure.